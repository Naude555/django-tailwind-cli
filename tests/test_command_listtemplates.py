from typing import Any

from django.core.management import call_command


def test_listtemplates(theme_app_path: Any, snapshot: Any, capsys: Any):
    """`tailwind listtemplates` lists all available templates."""
    call_command("tailwind", "listtemplates")
    captured = capsys.readouterr()
    assert captured.out == snapshot
