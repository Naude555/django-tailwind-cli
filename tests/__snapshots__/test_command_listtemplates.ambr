# serializer version: 1
# name: test_listtemplates
  '''
  /Users/oa/Projekte/privat/django-tailwind-cli/src/django_tailwind_cli/templates/
  tailwind_cli/tailwind_css.html 
  /Users/oa/Projekte/privat/django-tailwind-cli/src/django_tailwind_cli/templates/
  tailwind_cli/base.html
  
  '''
# ---
